config/localTemplate.go

```diff
@@ -318,7 +318,8 @@ type Local struct {
 
 	// CatchpointTracking determines if catchpoints are going to be tracked. The value is interpreted as follows:
 	// A value of -1 means "don't track catchpoints".
-	// A value of 1 means "track catchpoints as long as CatchpointInterval is also set to a positive non-zero value". If CatchpointInterval <= 0, no catchpoint tracking would be performed.
+	// A value of 1 means "track catchpoints as long as CatchpointInterval > 0".
+	// A value of 2 means "track catchpoints and always generate catchpoint files as long as CatchpointInterval > 0".
```

Comment:
I recognize this mode was supported since at least unlimited assets, but what is the use-case for setting this to `2` / forcing catchpoint file generation when a node is non-archival? I.E. if for testing, wondering if there are more general ways to handle that.

Reply:
The use case is mainly for testing, in case you want to generate catchpoints but not have to keep a deep block history of all blocks.